assets/js/modules/adsense/index.js

```diff
@@ -84,6 +75,12 @@ domReady( () => {
 			settingsEditComponent: SettingsEdit,
 			settingsViewComponent: SettingsView,
 			setupComponent: SetupMain,
+			checkRequirements: () => {
+				const isAdBlockerActive = select( STORE_NAME ).isAdBlockerActive();
+				const errorMessage = __( 'Ad blocker detected, you need to disable it in order to set up AdSense.', 'google-site-kit' );
+
+				return ! isAdBlockerActive ? true : { adsense: errorMessage };
```

Comment:
See above, this should return `true` or an error object.

Reply:
Updated.